### ISSUE TITLE
Noise

### ISSUE THREAD

**Issue by Prior99** [Rantanen/node-opus#1]
When running the test 

```
prior@cronosl ~/P/n/n/n/test> node app.js
Wrote following files.
in.pcm   : Input PCM signal
out.opus : OPUS encoded input signal
out.pcm  : Decoded PCM signal
```

It generates the `in.pcm` and `out.pcm` files as suspected.
However, when playing them using mplay, the `out.pcm` has noise while the `in.pcm` has not:

```
mplayer -rawaudio samplesize=4:channels=2:rate=48000 -demuxer in.pcm

mplayer -rawaudio samplesize=4:channels=2:rate=48000 -demuxer out.pcm
```

Also playing the `out.opus` indicates that the files duration is 0 seconds. (Mplay immediatly reaches end of file), however, the file is not empty.

```
prior@cronosl ~/P/n/n/n/test> ls -lAh out.opus
-rw-r--r-- 1 prior prior 67K Apr 28 11:29 out.opus
prior@cronosl ~/P/n/n/n/test> mplayer out.pcm
MPlayer SVN-r37379 (C) 2000-2015 MPlayer Team
210 audio & 441 video codecs
do_connect: could not connect to socket
connect: No such file or directory
Failed to open LIRC support. You will not be able to use your remote control.

Playing out.pcm.
libavformat version 56.25.101 (internal)


Exiting... (End of file)
```

**Comment by Rantanen** [comment]
Yeah, there seems to be something funky with the encoder output. I noted it in quick tests last night but didn't have time to examine it further.

The "opus" file is raw opus frames so doubt anything plays it pack as it is. Usually opus audio is delivered inside the ogg container.

**Comment by Rantanen** [comment]
Fixed in 8bd9d2df81c667f6504b135e00b669d85405f9e4

Decoded output length was wrong at least. We were skipping half the output which just happened to match the sine wave almost.
